src/config/rcxml.c

```diff
@@ -1343,6 +1446,8 @@ rcxml_finish(void)
 	current_mouse_context = NULL;
 	current_keybind_action = NULL;
 	current_mousebind_action = NULL;
+	current_child_action = NULL;
```

Comment:
We must free the action-queries + child-actions here.

Reply:
They are added to the parent action's list now. Or did I miss some error case?

Reply:
I don't see the new actions being freed nor the queries being free'd in `action_free()` / `action_list_free()`, similar to the `if (arg->type == LAB_ACTION_ARG_STR) {` condition in `action_free()`.

Reply:
Hivemind! Just mentioned it further.

Reply:
After discussion on IRC my question regarding the list for `struct view_query` is answered: To be able to implement OR logic when multiple queries share the same parent.